language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/AbstractItemResolver.java

```diff
@@ -237,4 +228,81 @@ protected boolean isAnnotationStart(LSServiceOperationContext ctx) {
             return this.getCompletionItemList(either.getRight());
         }
     }
+
+    /**
+     * Get the completion item for a package import.
+     * If the package is already imported, additional text edit for the import statement will not be added.
+     * 
+     * @param ctx               LS Operation context
+     * @return {@link List}     List of packages completion items
+     */
+    protected List<CompletionItem> getPackagesCompletionItems(LSServiceOperationContext ctx) {
+        // First we include the packages from the imported list.
+        List<String> populatedList = new ArrayList<>();
+        List<CompletionItem> completionItems = CommonUtil.getCurrentFileImports(ctx).stream()
+                .map(bLangImportPackage -> {
+                    String orgName = bLangImportPackage.orgName.toString();
+                    String pkgName = String.join(".", bLangImportPackage.pkgNameComps.stream()
+                            .map(id -> id.value)
+                            .collect(Collectors.toList()));
+                    CompletionItem item = new CompletionItem();
+                    item.setLabel(orgName + "/" + pkgName);
```

Comment:
Just a suggestion,.. let's move "/" to a const or a global var as it is used in multiple location in different functions ?